.gitpod.Dockerfile

```diff
@@ -66,6 +66,9 @@ RUN sudo apt-get update \
 # Source of original file https://github.com/gitpod-io/website/blob/main/src/routes/docs/languages/javascript.md
 
 # Use nvm to setup the version of node we want to use.
-RUN bash -c ". .nvm/nvm.sh     && nvm install 14     && nvm use 14     && nvm alias default 14"
+RUN bash -c ". .nvm/nvm.sh \
+    && nvm install \
+    && nvm use \
+    && npm install -g npm@8"
 
```

Comment:
Can you try if it works ok with this updated command?

Specifying node version shouldn't be necessary since that's already done in `.nvmrc`, and upgrading npm itself should be as simple as the last command.

Reply:
It's not working at all now:

```bash
No .nvmrc file found

Node Version Manager (v0.39.0)

Note: <version> refers to any version-like string nvm understands. This includes:
  - full or partial version numbers, starting with an optional "v" (0.10, v0.1.2, v1)
  - default (built-in) aliases: node, stable, unstable, iojs, system
  - custom aliases you define with `nvm alias foo`
```

```bash
Error: build failed: cannot build base image: The command '/bin/sh -c bash -c ". .nvm/nvm.sh     && nvm install     && nvm use     && npm install -g npm@8"' returned a non-zero code: 127
```

I think there's a problem with finding the `.nvmrc` file?

Reply:
Ah yeah, I might need to copy that file into the image. I'll give it a try later.